drivers/hmis/spec/models/hmis/auto_exit_config.rb

```diff
@@ -1,58 +0,0 @@
-###
```

Comment:
GH is marking this file as deleted, but it was moved here: `drivers/hmis/spec/models/hmis/project_auto_exit_config_spec.rb`
The first test case is unchanged, and then `it 'should throw errors if the auto-exit is not configured properly' do` was added 🐥